src/lib/project-loader-hoc.jsx

```diff
@@ -44,7 +44,7 @@ const ProjectLoaderHOC = function (WrappedComponent) {
                         analytics.event({
                             category: 'project',
                             action: 'Load Project',
-                            value: projectId,
+                            label: projectId,
```

Comment:
Is this name change significant? e.g. something (not in this PR) is referring to label and this was a bug fix? Or is this just a clarification change?

Reply:
Sorry it is in the PR description, but `value` is only for numeric data in GA and only used for summing, we want the `label` field which allows you to search/list by those properties